src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitRadioInput.js

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useState, Fragment } from 'react';
```

Comment:
This is a component and can be placed in the `components` subfolder.

Reply:
I don't think it's flexible enough yet to live there..